sdk/core/amqp-common/rollup.base.config.js

```diff
@@ -141,7 +141,7 @@ export function browserConfig(test = false) {
   };
 
   if (test) {
-    baseConfig.input = "dist-esm/test/**/*.spec.js";
+    baseConfig.input = "dist-esm/test/*.spec.js";
```

Comment:
Entry points for node tests at line 67 have to be updated as well.

Reply:
![image](https://user-images.githubusercontent.com/10452642/78188362-f10c9600-7424-11ea-8841-9bcef8bea3d7.png)

```js
[
      "dist-esm/test/*.spec.js",
      "dist-esm/test/node/*.spec.js",
]
```

Reply:
The node rollup config does actually pull in everything now as expected (since there are no browser-specific tests) but I can make this change.

Reply:
That's new, I did not expect that.

Reply:
I was getting tsc warnings that `input` can't be an array so made a fancy string instead.